docs/source/slurmguide/slurmarray.md

```diff
@@ -73,42 +60,12 @@ Lastly, we usually use job arrays for embarrassingly parallel jobs. If your case
 #SBATCH --mem-per-cpu=2G ## debug prior to know how much RAM
 :::
 
-:::{warning}
-50 is the maximum number of jobs allowed to be run at once per user-account.
-:::
+## Job Arrays from Commmand Line
 
-The above examples apply for interactive mode, as well. For instance:
+You can launch job arrays from the sbatch submission on the command line as follows:
 
 :::{code} bash
 sbatch --array=<indexes> [options] script_file
 :::
 
-Indexes can be listed as `1-5` (i.e., one to five), `=1,2,3,5,8,13` (i.e., each index listed), or `1-200%5` (i.e., produce a 200 task job array with only 5 tasks active at any given time). **The symbol used is the % sign, which tasks to be submitted at once** (again, cannot be set larger than 50).
-
-
-### Use-cases: Job Arrays
-Job arrays can be used in situations where you have to process multiple data files using the same procedure or program. Instead of creating multiple scripts or running the same script multiple times, you can create a job array, and Slurm will handle the parallel execution for you.
-
-### Example using Job Array Flag
-
-In the following script, the `$SLURM_ARRAY_TASK_ID` variable is used to differentiate between array tasks.
-
-:::{code} bash
-#!/bin/bash
-#SBATCH -J MyArrayJob           # Job name
-#SBATCH -N 1                    # Number of nodes
-#SBATCH -n 1                    # Number of tasks
-#SBATCH -o output_%A_%a.txt     # Standard output file (%A for array job ID, %a for array index)
-#SBATCH -e error_%A_%a.txt      # Standard error file
-
-# Your program/command here
-srun ./my_program input_$SLURM_ARRAY_TASK_ID
-:::
-
-To submit this job array, save it as `my_array_job.sh` and run:
-
-:::{code} bash
-sbatch --array=1-50 my_array_job.sh
-:::
-
-This command will submit 50 jobs, running `my_program` with `input_1` through `input_100`.
+Indexes can be listed as `1-5` (i.e., one to five), `=1,2,3,5,8,13` (i.e., each index listed), or `1-200%5` (i.e., produce a 200 task job array with only 5 tasks active at any given time).
```

Comment:
```suggestion
Indexes can be listed as `1-5`, `=1,2,3,5,8,13` (i.e., each index listed), or `1-200%5` (i.e., produce a 200 task job array with only 5 tasks active at any given time).
```
This sentence is a bit hard to read with all the parenthesis, but also a bit backwards if that makes sense? Like the info in the parenthesis should be switched with the numbers?